bom/openhab-addons/pom.xml

```diff
@@ -14,952 +13,967 @@
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.bosesoundtouch</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.buienradar</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.chromecast</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.cm11a</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.coolmasternet</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.daikin</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.darksky</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.deconz</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.denonmarantz</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.digiplex</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
-      <groupId>org.openhab.binding</groupId>
+      <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.digitalstrom</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.dlinksmarthome</artifactId>
-      <version>${project.version}</version>
+      <version>2.5.0-SNAPSHOT</version>
```

Comment:
Could we also use the project version some way? It might make releasing harder? What is the plan should this file stay in the repo and be comitted after running the goal manual (like the copyright headers?)

Or is there some other plan?

Reply:
version: Not so easy, I'll try. The problem is that maven replaces the place-holder in l. 72 of the pom before replacing. I don't know how to prevent that.

I would prefer to run it on each `clean install` (i.e. re-build it every time) and remove it from the repo. But maven will not start at all if the bom-pom is missing.

Reply:
Which is strange, because the BOM is not used within this repo and is really just a generated file to be uploaded to bintray.

Reply:
https://github.com/openhab/openhab2-addons/blob/169cd570b397d057b450bb86bc5ac50446e94bf7/pom.xml#L768

Is the culprit. This includes the bom/pom.xml which includes the openhab-addons/pom.xml. If we remove it from the repo this is missing and maven complains.